apps/cow-fi/styles/global.styles.ts

```diff
@@ -1,4 +1,4 @@
-import styled, { createGlobalStyle } from 'styled-components'
```

Comment:
We had a lint rule before to prevent using 'styled-components', maybe doesn't affect cow.fi?